src/Expression/ForClasses/IsOneOf.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Arkitect\Expression\ForClasses;
+
+use Arkitect\Analyzer\ClassDescription;
+use Arkitect\Expression\Description;
+use Arkitect\Expression\Expression;
+use Arkitect\Rules\Violation;
+use Arkitect\Rules\ViolationMessage;
+use Arkitect\Rules\Violations;
+
+final class IsOneOf implements Expression
+{
+    /** @var array<class-string> */
+    private array $allowedFqcnList;
+
+    /**
+     * @param class-string ...$allowedFqcnList
+     */
+    public function __construct(string ...$allowedFqcnList)
```

Comment:
Can we avoid variadic parameter?
We removed it from other expression to add the exclude parameter if needed

Reply:
Sure.
I dropped the commit that renamed the expression, and applied the previous suggestion where we only accept one fqcn to test, instead of the variadic.

let me know if anything else is needed from my side.